apps/web/src/routes/b/+page.svelte

```diff
@@ -561,6 +562,33 @@
     isTrackerPaused$.next(wasTrackerPaused);
   }
 
+  async function handleJump() {
+    const target = await new Promise((resolver) => {
```

Comment:
1. please add types to the promise result
2. for people having the tracker enabled this can lead to data spikes - better to autopause before and unpause after (see other code in the same file)